modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -1020,15 +1020,23 @@ const FDSEntries = ({
 						data: {
 							id: 'view',
 						},
-						icon: 'view',
-						label: Liferay.Language.get('view'),
+						icon: 'pencil',
+						label: Liferay.Language.get('edit'),
 						onClick: onViewClick,
 					},
 					{
-						icon: 'pencil',
+						type: "group",
+						separator: true,
+					},
+					{
+						icon: 'blank',
 						label: Liferay.Language.get('rename'),
 						onClick: onRenameClick,
 					},
+					{
+						type: "group",
+						separator: true,
```

Comment:
I'm surprised SF didn't complain about this, but can we alphabetically sort `separator` and `type`?